wwwapp/management/commands/download_remote_images.py

```diff
@@ -0,0 +1,61 @@
+import hashlib
+import mimetypes
+import os.path
+import re
+import requests
+from urllib.parse import urljoin
+
+from django.core.management.base import BaseCommand
+
+import wwwapp.settings as settings
+from wwwapp.models import Workshop, Article
+
+
+class Command(BaseCommand):
+    help = 'Download all images referenced in <img> tags in old descriptions and upgrade to storing them on our server'
+
+    def add_arguments(self, parser):
+        group = parser.add_mutually_exclusive_group(required=True)
```

Comment:
It should be possible to specify in parser that one of `force` and `dryrun` is required

Reply:
Good point

Reply:
Actually... that is the whole point of this being a `mutually_exclusive_group` XD That's why the condition in handle() is an assert and not an exception or error message